mk/vs/LightPcapNg.vcxproj.template

```diff
@@ -166,14 +179,17 @@
       <Optimization>Full</Optimization>
       <FunctionLevelLinking>true</FunctionLevelLinking>
       <IntrinsicFunctions>true</IntrinsicFunctions>
-      <PreprocessorDefinitions>WINx64;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
-      <AdditionalIncludeDirectories>$(PcapPlusPlusHome)\3rdParty\LightPcapNg\LightPcapNg\include;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
+      <PreprocessorDefinitions>PUT_USE_ZSTD_HERE;WINx64;NDEBUG;_LIB;%(PreprocessorDefinitions)</PreprocessorDefinitions>
+      <AdditionalIncludeDirectories>$(PcapPlusPlusHome)\3rdParty\LightPcapNg\LightPcapNg\include;PUT_ZSTD_INCLUDE_PATH_HERE;%(AdditionalIncludeDirectories)</AdditionalIncludeDirectories>
```

Comment:
why do we need those here? Can't we use the definitions in `PcapPlusPlusPropertySheet.props`?

Reply:
You can, I just defines it for all the projects then instead of only this one.

Anyways I did it.

Reply:
Well after making my comment below, please re-affirm this is what you want. Defining this in every project is less of a big deal than marking ZStd as a dependency for every project.

I have the code changes ready I just have not pushed them yet.

Reply:
Got it, that makes sense. for `PUT_USE_ZSTD_HERE` I agree, but for `PUT_ZSTD_INCLUDE_PATH_HERE` I'm not sure we need that, we might put `$(ZStdHome)\include` instead, same as you did in `<LibraryPath>$(ZStdHome)\dll;...`

Reply:
I didn't even notice you had highlighted the include part part as it was off the side of the window.  Ill see about cleaning that up on Monday.